app/src/main/java/com/isc/hermes/MainActivity.java

```diff
@@ -100,6 +104,7 @@ protected void onCreate(Bundle savedInstanceState) {
         addIncidentGeneratorButton();
         MarkerManager.getInstance(this).removeSavedMarker();
         launcher = createActivityResult();
+        initShowIncidentsController();
         testPolyline(); // this is a test method that will be removed once the functionality has been verified.
```

Comment:
If the comment is important for a TODO and if not delete it

Reply:
that's a work implemented from another teammate, I don't know the consequences about detete it.

Reply:
then just put it as TODO